src/css/login.css

```diff
@@ -1,9 +1,46 @@
 .login-text-box {
   width: 250px;
   height: 50px;
```

Comment:
might be worth cleaning these up later.  Seems like a lot of classes for this right now